pkg/storaged/client.js

```diff
@@ -969,42 +1074,56 @@ client.stratis_start = () => {
                     return true;
                 });
             });
-};
+}
 
 function stratis_fetch_properties(proxy, props) {
     const stratis = client.stratis_manager.client;
 
-    stratis.call(proxy.path, "org.storage.stratis2.FetchProperties.r4", "GetProperties", [props])
+    return stratis.call(proxy.path, "org.storage.stratis2.FetchProperties.r4", "GetProperties", [props])
```

Comment:
This most certainly doesn't work with stratis3, and https://github.com/stratis-storage/stratisd/pull/2817 seems to even get rid of this custom property API (yay!). How can this not check for stratis3? This function is used in a lot of places, are they all called with stratis2 only, and stratis3 somehow does not need all this shuffling? If so, please rename them to `stratis2_*`.

Reply:
> If so, please rename them to stratis2_*.

Yes, that's a good point.  All this code is for Stratis 2 only. It will all go away with Stratis 3, and that's the reason why I have advocated to get rid of FetchProperties. I am very happy that it is happening. :-)

Reply:
Yay, happy to hear! So yes, let's rename all of these to `stratis2_*` then

Reply:
Done.